net-proxy/3proxy/3proxy-0.7.1.2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+EAPI=5
+inherit eutils
+
+DESCRIPTION="really tiny cross-platform proxy servers set"
+HOMEPAGE="http://www.3proxy.ru/"
+SRC_URI="http://3proxy.ru/${PV}/${P}.tgz"
+LICENSE="GPL3"
```

Comment:
This is incorrect. Please match the license to a proper license in /usr/portage/licenses. I guess it would be `GPL-3` or `GPL-3+`. And please run `repoman full` before submitting ebuilds.